code/modules/mob/living/simple_animal/hostile/syndicate_mobs.dm

```diff
@@ -308,12 +303,8 @@
 		speed = 2
 		projectiletype = /obj/item/projectile/bullet/sniper/penetrator // Ignores cover.
 		projectilesound = 'sound/weapons/gunshots/gunshot_sniper.ogg'
-		loot = list(/obj/effect/mob_spawn/human/corpse/syndicatequartermaster,
-					/obj/item/salvage/loot/syndicate,
-					/obj/effect/decal/cleanable/blood/innards,
-					/obj/effect/decal/cleanable/blood,
-					/obj/effect/gibspawner/generic,
-					/obj/effect/gibspawner/generic)
+		loot -= /obj/item/salvage/loot/syndicate // I don't like this, but it guarantees the stronger version always drop 1 salvage.
+		loot += /obj/item/salvage/loot/syndicate // First remove any salvage that might appear due to the parent call, then make it drop a salvage on death. Prevents 2 salvage.
```

Comment:
```suggestion
		if(!(/obj/item/salvage/loot/syndicate in loot))
			loot += /obj/item/salvage/loot/syndicate
```
Try this

Reply:
Couldn't you just |= instead of all this?

Reply:
Imo kinda cursed but that should work yeah